cypress/e2e/Homeworks/hw13-forms/register_new_account.cy.js

```diff
@@ -0,0 +1,47 @@
+import { HomePage } from "../../../page-objects/hw13-form/home_page";
+//import { faker } from "@faker-js/faker";
+import { fakerCS_CZ as faker } from "@faker-js/faker";
+
+describe("Register a New Account", () => {
+  it("Fill out the form and register a new account", () => {
+    const czechPhone = faker.phone.number("+420#########");
+    const loginName = `Bramburka_${faker.number.int({
+      min: 1000,
+      max: 99999,
+    })}`;
+
+    new HomePage()
+      .openAutomationteststore()
+      .loginOrRegisterButtonIsVisible()
+      .loginOrRegisterButtonHaveText("Login or register")
+      .clickLoginOrRegisterButton()
+      .continueButtonIsVisible()
```

Comment:
Toto není chyba, ale kontrolní metody můžeš sloučit s ostatními akcemi (zkontroluješ viditelnost a klikneš). Cypress si navíc viditelnost hlídá při interakci s elementem.